scripts/bootstrap.sh

```diff
@@ -1,10 +1,10 @@
-yum install -y wget
-wget https://bootstrap.pypa.io/get-pip.py
-python get-pip.py
-
-pip install azure-keyvault
-pip install pyopenssl
-
-tar -xzf vm_scripts.tar.gz
-cd scripts
-python download_certificate.py
\ No newline at end of file
+yum install -y wget
+wget https://bootstrap.pypa.io/get-pip.py
+python get-pip.py
+
+pip install azure-keyvault
+pip install pyopenssl
+
+tar -xzf vm_scripts.tar.gz
+cd scripts
+python download_certificate.py &
```

Comment:
Should this trailing & be here?

Reply:
Yes because you are making it a background task